### ISSUE TITLE
google_maps_flutter 0.5.13 app bar flash when map inside tab view

### ISSUE THREAD

**Issue by spazemantheone** [flutter/flutter#32201]
![flutter_map_flash 2019-05-07 08_35_48](https://user-images.githubusercontent.com/9468324/57305362-8fbe3480-70a6-11e9-81e2-c6864b9ac413.gif)

## Steps to Reproduce

```dart
 @override
  Widget build(BuildContext context) {
    _buildMarkers();
    final GoogleMap map = GoogleMap(
      initialCameraPosition: CameraPosition(
          target: LatLng(results.latitude, results.longitude), zoom: 14),
      onMapCreated: _onMapCreated,
      myLocationEnabled: true,
      markers: Set<Marker>.of(markers.values),
    );

    return Scaffold(
      appBar: MyAppBar(true).build(context),
      body: Container(
        child: Column(
          mainAxisAlignment: MainAxisAlignment.start,
          mainAxisSize: MainAxisSize.max,
          children: <Widget>[
            Center(
                child: Padding(
                    padding: EdgeInsets.fromLTRB(10.0, 20, 10, 10),
                    child: Column(children: [
                      Text("Showing surcharge-free TransFund ATMs"),
                      Text(
                          " within ${results.radius} miles of ${results
                              .searchCriteria}.",
                          style: TextStyle(fontWeight: FontWeight.bold)),
                      Row(
                          mainAxisAlignment: MainAxisAlignment.center,
                          children: [
                            Center(
                                child: FlatButton(
                                  // padding: EdgeInsets.fromLTRB(10, 0, 0, 10),

                                    child: Row(
                                      children: [
                                        new Image.asset("graphics/mappin.png"),
                                        Text("Change location",
                                            style: TextStyle(
                                                color: const Color(0xff3964BF)))
                                      ],
                                    ),
                                    onPressed: () {
                                      Navigator.pop(context);
                                    }))
                          ]),
                    ]))),

            TabBar(
              controller: _tabController,
              labelColor: const Color(0xffB00027),
              tabs: [
                Tab(
                    child: Text('List View'),
                    icon: ImageIcon(
                      new AssetImage("graphics/listviewred-2x.png"),
                      size: 0,
                    )),
                Tab(
                    child: Text('Map View'),
                    icon: ImageIcon(
                      new AssetImage("graphics/mapviewred-2x.png"),
                      size: 0,
                    )),
              ],
            ),
            Expanded(
              flex: 1,
              child: TabBarView(
                controller: _tabController,
                physics: NeverScrollableScrollPhysics(),
                children: <Widget>[

                  CustomScrollView(
                    shrinkWrap: true,
                    slivers: <Widget>[
                      SliverPadding(
                        padding: const EdgeInsets.all(10.0),
                        sliver: SliverList(
                          delegate:
                          SliverChildListDelegate(_buildLocationList()),
                        ),
                      ),
                    ],
                  ),
                map
                ],
              ),
            ),
          ],
        ),
      ),
    );
  }
```

## Logs
```
[✓] Flutter (Channel stable, v1.2.1, on Mac OS X 10.14.4 18E226, locale en-US)
    • Flutter version 1.2.1 at /Development/flutter-1.0
    • Framework revision 8661d8aecd (3 months ago), 2019-02-14 19:19:53 -0800
    • Engine revision 3757390fa4
    • Dart version 2.1.2 (build 2.1.2-dev.0.0 0a7dcf17eb)

[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/caseymiller/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling
      support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_152-release-1248-b01)
    • All Android licenses accepted.

[✓] iOS toolchain - develop for iOS devices (Xcode 10.2.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.2.1, Build version 10E1001
    • ios-deploy 1.9.4
    • CocoaPods version 1.5.3

[!] Android Studio (version 3.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build
      1.8.0_152-release-1248-b01)

[✓] IntelliJ IDEA Ultimate Edition (version 2018.3.5)
    • IntelliJ at /Applications/IntelliJ IDEA.app
    • Flutter plugin version 33.3.2
    • Dart plugin version 183.5912.10
```

```
```

```
```

```
```

**Comment by janmoppel** [comment]
Hello @spazemantheone !

Did you manage to solve the issue? Does the problem still persist in the latest stable version?

**Comment by spazemantheone** [comment]
Fixed in flutter 1.7 I believe.

**Comment by iapicca** [comment]
Hi @spazemantheone thanks for notifying this
